daemon/inertiad/webhook/github.go

```diff
@@ -18,3 +19,19 @@ func parseGithubEvent(rawJSON map[string]interface{}, event string) (Payload, er
 		return nil, errors.New("unsupported Github event")
 	}
 }
+
+// get payload bytes from request body
+func getGithubPayloadBytes(contentType string, body []byte) ([]byte, error) {
+	switch contentType {
+	case "application/x-www-form-urlencoded":
+		values, err := url.ParseQuery(string(body))
+		if err != nil {
+			return nil, err
+		}
+		return []byte(values.Get("payload")), nil
+	case "application/json":
+		return body, nil
+	default:
+		return nil, errors.New("Github Webhook Content-Type must be application/json or x-www-form-urlencoded")
```

Comment:
"Github webhook Content-Type must be application/json or **application**/x-www-form-urlencoded" 👍

Reply:
done thanks !